src/Controller/HomepageController.php

```diff
@@ -23,9 +23,12 @@ class HomepageController extends AbstractController
      */
     public function homepage(Request $request): Response
     {
-        $form = $this->createForm(TypoFixerType::class);
-        $form->handleRequest($request);
+        $defaults = [
```

Comment:
`$data`

Reply:
For sure I will but : 
https://symfony.com/doc/current/components/form.html#setting-default-values

:stuck_out_tongue:

Reply:
oh, you wanna play?

OK 👼🏼 

* https://github.com/symfony/symfony/blob/5.x/src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php#L325-L327
* https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/Form/FormFactory.php#L26-L29

https://github.com/symfony/symfony-docs/edit/4.4/components/form.rst => I let you open a PR to fix the doc :)